src/services/JobClusterService.ts

```diff
@@ -50,3 +50,28 @@ export async function fetchJobClusters(
 
   return jobClusters;
 }
+
+export async function fetchJobClusterByName(
+  regionEnvs: EnvRegion[],
+  clusterName: string,
+): Promise<Cluster | null> {
```

Comment:
It is not necessary but it would be clear to a developer to know what is being returned, who is working with function. I would remove this.
Also I am learning alot while contributing to mantis ui, so tend to do some mistakes initially but later figure them out.

Reply:
Yup, lots of these things are up for debate. No right or wrong answer. I just think with the typescript tooling, hovering over the return type does the job of knowing what is being returned.
I tend to follow this philosophy: https://www.youtube.com/watch?v=nwSe95uFN8E&ab_channel=MattPocock

Reply:
That being said if its a really complicated function and the typescript type system does not do a good job of inference, then by all means add an explicit type.

Reply:
> Yup, lots of these things are up for debate. No right or wrong answer. I just think with the typescript tooling, hovering over the function itself does the job of knowing what is being returned implicitly. I tend to follow this philosophy: [youtube.com/watch?v=nwSe95uFN8E&ab_channel=MattPocock](https://www.youtube.com/watch?v=nwSe95uFN8E&ab_channel=MattPocock)

Yes, this makes total sense. Would try to avoid giving types if unnecessary. 
Committed the changes you can review them.